src/installer/pkg/sfx/Microsoft.NETCore.App/Directory.Build.props

```diff
@@ -161,7 +161,7 @@
     <PlatformManifestFileEntry Include="api-ms-win-crt-utility-l1-1-0.dll" IsNative="true" FallbackFileVersion="$(WindowsForwarderFileVersion)" />
     <PlatformManifestFileEntry Include="API-MS-Win-core-xstate-l2-1-0.dll" IsNative="true" FallbackFileVersion="$(WindowsForwarderFileVersion)" />
     <PlatformManifestFileEntry Include="ucrtbase.dll" IsNative="true" />
-    <PlatformManifestFileEntry Include="msquic.dll" IsNative="true" />
+    <PlatformManifestFileEntry Include="msquic.dll" IsNative="true" FallbackFileVersion="$(WindowsForwarderFileVersion)" />
```

Comment:
This cannot possibly be the correct file version.  This is a completely different source.  Maybe it needs it's own property?

Reply:
My care-level for getting this perfect is rather low.  The side effects of having it too low just mean a source-built ref-pack might accidentally include msquic.dll with the app (which we don't even OOB).  The runtime would still make the right choice.

I recommend just adding a new property and giving it a value that's <= the current file version we restore.